clients/privacy-center/config/config.json

```diff
@@ -30,6 +30,7 @@
   ],
   "includeConsent": true,
   "consent": {
+    "cookieName": "ethyca_consent",
```

Comment:
Open Question: Should the cookie name be configurable? 

So much of the privacy center is configurable, this seemed like a good idea. However, I could also see this as introducing too many moving parts.